src/SourceBuild/content/build.proj

```diff
@@ -25,6 +27,17 @@
              StopOnFirstFailure="true" />
   </Target>
 
+  <Target Name="AfterBuild" AfterTargets="Build">
```

Comment:
Make this name more precise, i.e. `PublishVerticalManifest`. AfterBuild could be a reserved msbuild target name.

Reply:
fixed

Reply:
I think I was hitting some reserved target name with it, windows stopped failing after the change